README.md

```diff
@@ -44,15 +44,11 @@ Kivy ships with many examples which can be found in the `examples` folder.
 Support
 -------
 
-If you need assistance, you can ask for help on our mailing list:
+If you need help, please feel free to seek assistance through our support channels:
 
-* User Group : https://groups.google.com/group/kivy-users
-* Email      : kivy-users@googlegroups.com
+Join the  [Kivy Users Google Group](https://groups.google.com/group/kivy-users).
+Participate in discussions on [Discord](https://chat.kivy.org).
```

Comment:
Nice work 👍 , while we are at it, i think its ok if we also add link to our all new [discussion](https://github.com/orgs/kivy/discussions) threads here on GH

```suggestion
- Join the [Kivy Users Google Group](https://groups.google.com/group/kivy-users).
- Ask questions using [discussions](https://github.com/orgs/kivy/discussions/new?category=q-a) on github
- Use support-channels on our official [Discord](https://chat.kivy.org) server.  
```

Reply:
Sure

Reply:
@seharbat00l updates on this?